Jenkinsfile

```diff
@@ -92,8 +157,13 @@ pipeline {
       when {
         expression { env.BRANCH_NAME == 'latest' }
       }
+      options {
+        // Do not perform the SCM step
+        skipDefaultCheckout true
+      }
       agent any
       steps {
+        unstash 'simorgh'
```

Comment:
Do we unstash here so the files are passed to the `simorgh-infrastructure/latest` job? I cant see this documented as a feature, do you have any links you can share?

Reply:
Yes that is what will happen. But not yet. I first have to add changes to simorgh-infrastructure for that to work.

Reply:
Do you have any docs to reflect that this will work?

Reply:
the static-deployment pipeline does this already

Reply:
@dr3 this same process is being used here https://github.com/bbc/simorgh-infrastructure/blob/latest/Jenkinsfile#L75-L86